scripts/globals/teleports.lua

```diff
@@ -151,6 +151,7 @@ local destinations =
     [ids.ESCHA_RUAUN]           = {   -0.371,  -34.279, -466.980, 192, 289 },
     [ids.MISAREAUX_CONFLUENCE]  = {  -57.385,  -21.460,  568.941, 160,  25 }
 }
+xi.teleport.destination = destinations
```

Comment:
If you're going to do this, make the initial local table global and update any later calls to `destinations` to use it